docs/usage.rst

```diff
@@ -333,11 +333,11 @@ And set an ``Access-Control-Allow-Origin`` HTTP header on that file.
 Promises
 --------
 
-By default, Raven.js does not capture unhandled promise rejections.
+By default, Raven.js capture unhandled promise rejections as described in official ECMAScript 6 standard.
 
-Most Promise libraries have a global hook for capturing unhandled errors. You will need to
-manually hook into such an event handler and call ``Raven.captureException`` or ``Raven.captureMessage``
-directly.
+Most Promise libraries however, have a global hook for capturing unhandled errors. You will need to disable default behaviour
+by setting ``captureUnhandledRejections`` option to ``false`` and manually hook into such an event handler
```

Comment:
> You will need to disable default behaviour
is there harm in leaving it?

Reply:
Not really, but it'll silence the error unless you set `Raven.debug = true;` to see the log.